lib/Runtime/PlatformAgnostic/Platform/Linux/NumbersUtility.cpp

```diff
@@ -0,0 +1,267 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+#include "Common.h"
+#include "ChakraPlatform.h"
+#include <locale.h>
+
+namespace PlatformAgnostic
+{
+namespace Numbers
+{
+    #define BufSize 256
+
+    template<class T, class N=T>
```

Comment:
I prepared this method for later use since It's needed for ICU part. I had implemented that too but wanted to keep separate due to issues with ICU versions.